src/xGadgetron/cGadgetron/gadgetron_data_containers.cpp

```diff
@@ -654,7 +654,7 @@ AcquisitionsFile::items()
 }
 
 void 
-AcquisitionsFile::get_acquisition(unsigned int num, ISMRMRD::Acquisition& acq)
+AcquisitionsFile::get_acquisition(unsigned int num, ISMRMRD::Acquisition& acq) const
```

Comment:
cannot be `const`: contains

	dataset_->readAcquisition(ind, acq);

and `readAcquisition(ind, acq)` is not `const`.